docs/development.md

```diff
@@ -82,15 +117,39 @@ sudo mv ./kind /usr/local/bin/kind
 kind cluster create
 ```
 
+### Using Dev Jump Box
+
+Because the resources deployed via the ARM template are locked down via Azure Firewall rules, a development VM has been provided as a "jump box". The main use for this dev box is to deploy resources within this subscription (e.g. deploying the K8s cluster). To connect to this machine, you may SSH into it using the same SSH key you created earlier.
+
+#### Dev Box Setup
+
+Currently the dev box is a standard Linux VM, so be sure to install the following:
+
+- [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest)
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-on-linux)
+- [Helm](https://helm.sh/docs/intro/install/)
+- [NPM](https://github.com/nodesource/distributions/blob/master/README.md)
+- [Docker](https://docs.docker.com/engine/install/ubuntu/#:~:text=%20Install%20from%20a%20package%20%F0%9F%94%97%20%201,the%20hello-world%20image.%0A%24%20sudo%20docker%20run...%20More%20) (if you will be building images on the box)
+
+Once those tools are installed, clone the SDS repo into that dev machine and continue with the `Configuration` section below.
+
 ### Configuration
 
 Update `deploy/helm/values.dev.yaml` with the values that match your environment. Specifically, you'll need
 
 * `worker.queueEndpoint`: Point to a queue in your Azure Storage Account
 * `identities.*`: Enter values for your own Service Principal(s). The `worker` identity will need the `Storage Queue Data Message Processor` (or higher) role on your queue
 
+>NOTE: If using the ARM template, managed identities have been created for you, and you can instead use the format found in `deploy/helm/values.yaml`, filling in the appropriate client IDs matching the pre-created identities.
+
 ### Install SDS
 
+If this is your first time using the K8s cluster, be sure that you have installed the AAD Identity Binding on the AKS cluster within the `kube-system` namespace.
+```shell
+kubectl apply -f https://raw.githubusercontent.com/Azure/aad-pod-identity/master/deploy/infra/deployment-rbac.yaml -n kube-system
```

Comment:
We should probably choose a known-good version versus the master branch

Reply:
Updated to use v1.6.0 release